index.html

```diff
@@ -64,132 +64,174 @@ <h1>
-                    <li>2015</li>
-                </ul>
-                <p class="card-main-text">
-                    A daily selection of privately personalized reads; no accounts
-                    or sign-ups required.
-                </p>
-                <ul class="technology-tags">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                </ul>
-                <button class="See-project-button" type="button">See Project</button>
+                    <h2 class="project-name">Tonic</h2>
+                    <ul class="project-frame">
+                        <h3>CANOPY</h3>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="card-main-text">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="technology-tags">
+                        <!-- <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li> -->
+                    </ul>
+                    <button class="See-project-button" type="button" value="0">See Project</button>
                 </div>
 
+             </div>
             </div>
+
+
+            <!-- Uncomment after debuggin;
             <div class="card-element card-reverse">
                 <img class="preview-picture" src="./img/Portfolio4.svg" alt="#">
                 <div class="text-box">
-                <h2 class="project-name">Multi-Post Stories</h2>
-                <ul class="project-frame">
-                    <h3>FACEBOOK</h3>
-                    <li>Full Stack Dev</li>
-                    <li>2015</li>
-                </ul>
-                <p class="card-main-text">
-                    A daily selection of privately personalized reads; no accounts
-                    or sign-ups required.
-                </p>
-                <ul class="technology-tags">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                </ul>
-                <button class="See-project-button" type="button">See Project</button>
+                    <h2 class="project-name">Multi-Post Stories</h2>
+                    <ul class="project-frame">
+                        <h3>FACEBOOK</h3>
+                        <li>Full Stack Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="card-main-text">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="technology-tags">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
```

Comment:
Great design team 🤝

- [ ] Kindly remove all commented code from your project as this is regarded as a bad practice.